### PR TITLE
Add Environment Variables Section

### PR DESCRIPTION
**Pull request by bvallelunga**
This PR adds a section for environment variables management.

### CODE REVIEW

**Comment by agamm** [comment]
Hi @bvallelunga in order to merge a new section we need to see at least 2 products in the subject, do you know another one?

**Comment by bvallelunga** [comment]
Hey @agamm there are a couple of other products very loosely in the space (secrets management) but nothing directly in our category. Can we make an exception for this one case?

File: README.md [hunk 1]
```diff
@@ -105,6 +106,11 @@ The definition of developer-first for this repo is:
 * [Yotpo](https://www.yotpo.com/platform/reviews/) - Reviews ratings and Q&A as a service.
 * [Gitter](https://gitter.im) - Chat and networking platform for developers.
 
+## Environment Variables
+
+*Manage environment variables.*
+* [Doppler](https://doppler.com/) - All your environment variables across projects and environments, in one place.
+
```
Comment (agamm) [inline README.md:113]:
```suggestion
* [Envkey](https://www.envkey.com/) - Environment management with syncing mechanism for all apps.
```

File: README.md [hunk 2]
```diff
@@ -105,6 +106,11 @@ The definition of developer-first for this repo is:
 * [Yotpo](https://www.yotpo.com/platform/reviews/) - Reviews ratings and Q&A as a service.
 * [Gitter](https://gitter.im) - Chat and networking platform for developers.
 
+## Environment Variables
```
Comment (agamm) [inline README.md:109]:
```suggestion
## Environment & Secret Management
```

File: README.md [hunk 3]
```diff
@@ -105,6 +106,11 @@ The definition of developer-first for this repo is:
...
+
+*Manage environment variables.*
```
Comment (agamm) [inline README.md:111]:
```suggestion
*Manage environment variables and secrets for multiple apps or projects.*
* [Vault](https://www.vaultproject.io/) -  API driven secret key management.
```

File: README.md [hunk 4]
```diff
@@ -105,6 +106,11 @@ The definition of developer-first for this repo is:
...
+* [Doppler](https://doppler.com/) - All your environment variables across projects and environments, in one place.
```
Comment (agamm) [inline README.md:112]:
```suggestion
* [Doppler](https://doppler.com/) - Environment variable management across projects, in one place.
* [1Password  Secrets Automation](https://1password.com/secrets/) - Secrets management with integration to 1Password.
```

File: README.md [hunk 5]
```diff
@@ -21,6 +21,7 @@ The definition of developer-first for this repo is:
 * [Databases & Spreadsheets](#databases--spreadsheets)
 * [Deployment Hosting](#deployment-hosting)
 * [Discussions](#discussions)
+* [Environment Variables](#environment-variables)
```
Comment (agamm) [inline README.md:24]:
```suggestion
* [Environment & Secret Management](#environment--secret-management)
```

**Review by agamm — CHANGES_REQUESTED** [review]
I added 2 more products that seem to be closely related.

File: README.md [hunk 6]
```diff
@@ -105,6 +106,13 @@ The definition of developer-first for this repo is:
 * [Yotpo](https://www.yotpo.com/platform/reviews/) - Reviews ratings and Q&A as a service.
 * [Gitter](https://gitter.im) - Chat and networking platform for developers.
 
+## Environment & Secret Management
+
+*Manage environment variables and secrets for multiple apps or projects.*
+* [Vault](https://www.vaultproject.io/) -  API driven secret key management.
```
Comment (agamm) [inline README.md:112]:
```suggestion
* [Vault](https://www.vaultproject.io/) - API driven secret key management.
```

File: README.md [hunk 7]
```diff
@@ -105,6 +106,13 @@ The definition of developer-first for this repo is:
...
+* [Doppler](https://doppler.com/) - Environment variable management across projects, in one place.
+* [1Password  Secrets Automation](https://1password.com/secrets/) - Secrets management with integration to 1Password.
```
Comment (agamm) [inline README.md:114]:
```suggestion
* [1Password Secrets Automation](https://1password.com/secrets/) - Secrets management with integration to 1Password.
```

**Review by agamm — APPROVED** [review]
Extra typos.

**Pull request merged**